R/app_server.R

```diff
@@ -30,6 +30,9 @@ app_server <- function(
     })
   check_appdata(app_data, meta)
   
+  session$userData$review_records <- reactiveValues()
+  session$userData$update_checkboxes <- reactiveValues()
```

Comment:
If I understand it correctly, the variable 'update_checkboxes' is only `TRUE`, `FALSE`, or `NULL`, based on the value of `input$form_reviewed` . Why is it called `update_checkboxes` and not `form_reviewed` or something similar, for consistency?

Reply:
Naming conventions are a bit in the eye of the beholder, but I named it `update_checkboxes` because that is the action the reactive is meant to trigger. Plus it would be inaccurate to call it `form_reviewed` because the value is more tied to changes in the sidebar checkbox, not to the review status of a form.